### PR TITLE
[RHPAM-3640] Configure Kafka JBPM Emitter from Operator

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/RHPAM-3640
Signed-off-by: desmax74 <mdessi@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by spolti** [comment]
@AntStephenson @almope hey guys, with this change, we need to keep an eye on these added jars, I kindly ask your help to remember us to update these jars before perform the release builds.

File: kieserver/modules/kieserver/module.yaml [hunk 1]
```diff
@@ -39,6 +39,9 @@ artifacts:
 - name: "kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00004.jar"
   dest: "/opt/kie/dependencies/jbpm-clustering"
   md5: "44f302877d28e2493fc28ead4dfb6dea"
+- name: "jbpm-event-emitters-kafka-7.52.0.Final-redhat-00007.jar"
```
Comment (spolti) [inline kieserver/modules/kieserver/module.yaml:42]: keep the fields on the same order that you have created for the clustering jar above.

**Review by spolti — CHANGES_REQUESTED** [review]

File: kieserver/modules/kieserver/module.yaml (hunk 1 above)
Comment (desmax74) [reply kieserver/modules/kieserver/module.yaml:42]: ok

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
hold for now, will merge everything together.

**Pull request merged**